app/params/supplejack_api/search_params.rb

```diff
@@ -17,7 +17,11 @@ class SearchParams < BaseParams
       :debug,
       :group_by,
       :group_order_by,
-      :group_sort
+      :group_sort,
+      :facet_range,
+      :facet_range_start,
+      :facet_range_end,
+      :facet_interval
```

Comment:
should tyhis be facet_*range*_interval?

Reply:
Good spotting bro! yes it should be :). Confusing that it works